torch/csrc/utils/python_dispatch.cpp

```diff
@@ -422,6 +422,17 @@ void initDispatchBindings(PyObject* module) {
             name, overload_name);
       });
 
+  m.def(
+      "_dispatch_maybe_import_abstract_impl_pystub", [](const char* qualname) {
+        // The abstract_impl API doesn't work with overload names yet.
```

Comment:
:0

Reply:
The `impl_{blah}` APIs are written in a spaghetti fashion because we originally wanted it to be a new python-only custom op API and then migrated them to work with C++ custom ops without actually doing a refactor.

I'm in the process of doing a refactor to get the `impl_{blah}` APIs to really look like "an API that sits directly above the low-level torch.library DispatchKey APIs"